lms/assets.py

```diff
@@ -24,12 +25,22 @@ def __init__(self, path, loader, auto_reload=False):
         :param auto_reload: If True, the parsed content is discarded if the
                             mtime of the file changes.
         """
-        self.path = path
+
+        self.path = self._find_file(path)
         self.loader = loader
         self._mtime = None
         self._cached = None
         self._auto_reload = auto_reload
 
+    @classmethod
+    def _find_file(cls, path):
+        path = os.path.abspath(os.path.join(resource_filename("lms", "."), "..", path))
+
+        if not os.path.isfile(path):
+            raise FileNotFoundError(f"Expected to find a file at '{path}'")
+
+        return path
```

Comment:
This was added to ensure we can run the tests from any directory, rather than just the root